mockssh/sftp.py

```diff
@@ -85,6 +85,16 @@ def stat(self, path):
         st = os.stat(path)
         return paramiko.SFTPAttributes.from_stat(st, path)
 
+    @returns_sftp_error
+    def lstat(self, path):
+        st = os.lstat(path)
+        return paramiko.SFTPAttributes.from_stat(st, path)
+
+    @returns_sftp_error
+    def symlink(self, src, dest):
+        os.symlink(src, dest)
```

Comment:
Why are you not wrapping it into try-except like `remove` and other functions down below?

Reply:
Good observation, @efiop, forgot about it :+1: